### PR TITLE
fix conversion

### PR DESCRIPTION
**Pull request by juanvallejo**
Followup to https://github.com/deads2k/origin/pull/43

an empty version in cmdutil.Factory#PrintResourceInfos was causing the `oc adm registry` config to not be marshaled correctly.

cc @deads2k

### CODE REVIEW

**Comment by juanvallejo** [comment]
(With these changes) currently seeing the following error from test-cmd:

```
FAILURE after 0.206s: hack/test-cmd.sh:109: executing 'oc new-project 'cmd-admin'' expecting success: the command returned the wrong error code
There was no output from the command.
Standard error from the command:
Error from server (InternalError): Internal error occurred: namespace "cmd-admin" not found
```

File: pkg/oc/cli/util/clientcmd/factory.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func (f *Factory) PrintResourceInfos(cmd *cobra.Command, isLocal bool, infos []*
 	}
 
 	printAsList := len(infos) != 1
-	object, err := AsVersionedObject(infos, printAsList, schema.GroupVersion{}, legacyscheme.Codecs.LegacyCodec())
+	object, err := AsVersionedObject(infos, printAsList, schema.GroupVersion{Version: "v1"}, legacyscheme.Codecs.LegacyCodec())
```
Comment (deads2k) [inline pkg/oc/cli/util/clientcmd/factory.go]: This smells wrong. Will reconsider tomorrow, but can you post before and after errors and output?

**Comment by juanvallejo** [comment]
@deads2k 
> This smells wrong. Will reconsider tomorrow, but can you post before and after errors and output?

Before:
Pastebin of error output: http://pastebin.test.redhat.com/539208
Fails here: https://github.com/openshift/origin/blob/master/hack/lib/start.sh#L633

After (with `schema.GroupVersion{Version: "v1"}` patch):
```
FAILURE after 0.162s: test/cmd/config.sh:28: executing 'env -u KUBERNETES_SERVICE_HOST oc get bc --user=""' expecting failure and text 'Missing or incomplete configuration info': the output content test failed
There was no output from the command.
Standard error from the command:
the server doesn't have a resource type "bc"
```
(Among a few others - tests keep running without exiting on a particular failure)

**Comment by juanvallejo** [comment]
Closing - conversion stuff addressed in https://github.com/deads2k/origin/pull/46
Everything else already part of rebase PR

**Pull request closed without merging**